spec/dummy/app/themes/default/views/thredded/shared/_topic_nav.html.erb

```diff
@@ -8,17 +8,19 @@
     </li>
   </ul>
 
-  <ul class="topic-navigation">
-    <li class="topic-navigation--active">
-      <%= link_to 'Topics', messageboard_topics_path(messageboard) %>
-    </li>
-    <li>
-      <%= link_to messageboard_private_topics_path(messageboard), class: "unread-#{unread_private_topics_count}" do %>
-        Private Topics 
-        <% unless unread_private_topics_count == 0 %>
-          <span class="topic-navigation--unread"><%= unread_private_topics_count %></span>
+  <% if messageboard %>
+    <ul class="topic-navigation">
+      <li class="topic-navigation--public">
+        <%= link_to 'Topics', messageboard_topics_path(messageboard) %>
+      </li>
+      <li class="topic-navigation--private">
+        <%= link_to messageboard_private_topics_path(messageboard), class: "unread-#{unread_private_topics_count}" do %>
+          Private Topics
+          <% unless unread_private_topics_count == 0 %>
+            <span class="topic-navigation--unread"><%= unread_private_topics_count %></span>
+          <% end %>
         <% end %>
-      <% end %>
-    </li>
-  </ul>
+      </li>
+    </ul>
+  <% end %>
```

Comment:
without this `if messageboard` conditional the main messageboard listing page pukes and goes belly up.